hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/schema/SqlMappingTest.java

```diff
@@ -165,10 +166,10 @@ private void test_alias(String javaClassName, String... aliases) {
 
     @Test
     public void test_caseInsensitiveType() {
-        sqlService.execute("CREATE MAPPING t1 TYPE TestStream");
-        sqlService.execute("CREATE MAPPING t2 TYPE teststream");
-        sqlService.execute("CREATE MAPPING t3 TYPE TESTSTREAM");
-        sqlService.execute("CREATE MAPPING t4 TYPE tEsTsTrEaM");
+        TestBatchSqlConnector.create(sqlService, "TestBatch", 1);
```

Comment:
This is case-insensitive mapping name, but the test was about case-insensitive mapping type.